keanu-project/src/main/java/io/improbable/keanu/tensor/intgr/IntegerTensor.java

```diff
@@ -66,6 +67,22 @@ static IntegerTensor scalar(int scalarValue) {
         return new ScalarIntegerTensor(scalarValue);
     }
 
+    /**
+     * @param toPile    an array of IntegerTensor
+     * @return  an IntegerTensor with toPile joined along a new dimension 0
```

Comment:
An example here would be helpful

Reply:
and/or an explanation of how this differs from `concat` (one creates an extra dimension, the other doesn't, right?)

Reply:
Yup. Sounds good.